pom.xml

```diff
@@ -134,6 +134,31 @@
             </resource>
         </resources>
 
+        <pluginManagement>
```

Comment:
Because mvn site is cpu intensive, slow and do stuff that we dont need for a a pull request integration.
The question should be, why not have a custom call ?